extractlua.py

```diff
@@ -0,0 +1,54 @@
+import sys, re, os, json
+from grammars.grammars import lua
+from extractor import find_subtree, remove_tabs_newlines
+
+def expr_filter(n):
+    return n.symbol.name == "explist"
+
+def func_filter(n):
+    if n.symbol.name == "stat" and n.children:
+        if n.children[0].symbol.name == "function":
+            return True
+        if n.children[0].symbol.name == "local" and n.children[2].symbol.name == "function":
+            return True
+    return False
+
+filters = {}
+filters["expressions"] = expr_filter
+filters["functions"] = func_filter
+
+RE_CMT = re.compile("--(.*)$")
+
+if __name__ == "__main__":
+    target = sys.argv[1]
+    output = sys.argv[2]
+    results = []
+    folder = "../../../lua/testes/"
```

Comment:
That is, I hope, just an unfortunate typo on the directory name. Otherwise the mind boggles.

Reply:
Unfortuntely, this is how it appears in the lua repo: https://github.com/lua/lua/tree/master/testes

Reply:
Ye gods! OK, let's hope that's a Portugese term!

Reply:
It seems this is the case.